python/ray/tests/test_stress.py

```diff
@@ -15,7 +15,7 @@ def ray_start_combination(request):
         initialize_head=True,
         head_node_args={
             "num_cpus": 10,
-            "redis_max_memory": 10**7
+            "redis_max_memory": 10**8
```

Comment:
OOC, why are we upping all of these? Is it because we have more resources per CI box.